_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,78 @@
+---
+title: New API Blueprint Tutorial and Examples
+excerpt: We've introduced a brand new API Blueprint tutorial along with some new examples.
+layout: post
+date: 2015-03-03 16:00:00 +0000
+author: kyle
+published: true
+comments: true
+---
+
+We've introduced a new [API Blueprint tutorial](http://apiary.io/blueprint) along with some brand new [API Blueprint examples](http://docs.pollsapi.apiary.io/). Our new tutorial walks you though the basics of API Blueprint along with how you can build your very first blueprint. It uses an example of a polls service, which allows consumers to view polls and vote in them.
```

Comment:
Since this blog post is introducing it I would say it shouldn't start with this sentence .. :)